data/MitAbhay.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Abhay Kumar Mittal",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Open Source | Devops | Full Stack Developer",
+  "avatar": "https://github.com/MitAbhay.png",
+  "tags": [
+    "Open Source",
+    "Javascript",
+    "Typescript",
+    "NextJS",
+    "DevOps"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/MitAbhay" },
+    { "icon": "FaGithub", "url": "https://github.com/MitAbhay" }
+  ],
+  "links": [
+    {
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/MitAbhay",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
```

Comment:
Not sure there's a color field for links. 🤔

Reply:
I think it's optional for links, but we have configured the color of more popular icons [here](https://github.com/EddieHubCommunity/LinkFree/blob/main/config/icons.json). So not required in this case!